sklearn/neighbors/typedefs.pxd

```diff
@@ -1,5 +1,6 @@
 #!python
 cimport numpy as np
+import numpy as np
```

Comment:
is a Python import valid in a ``pxd`` file? I thought that it doesn't actually do anything.